AI-and-Analytics/Getting-Started-Samples/IntelTensorFlow_GettingStarted/README.md

```diff
@@ -41,8 +41,27 @@ Third party program Licenses can be found here: [third-party-programs.txt](https
 
 ## Build and Run the Sample
 
-### Running Samples In DevCloud (Optional)
-If running a sample in the Intel DevCloud, please follow the below steps to build the python environment. Also, remember that you must specify the compute node (CPU, GPU, FPGA) and whether to run in batch or interactive mode. For more information, see the [Intel® oneAPI Base Toolkit Get Started Guide](https://devcloud.intel.com/oneapi/get-started/base-toolkit/) 
+<!---Include the next paragraph ONLY if the sample runs in batch mode-->
```

Comment:
Those are general contents for all samples.
Could you move out the contents from specific sample's README and move it to a general place such as README for all AI samples?

ex: https://github.com/oneapi-src/oneAPI-samples/blob/master/AI-and-Analytics/README.md